packages/zoe/src/contractFacet/types.js

```diff
@@ -191,7 +191,7 @@
 /**
  * @callback OfferHandler
  * @param {ZCFSeat} seat
- * @returns any
+ * @returns undefined
```

Comment:
I think this should be {unknown}

```suggestion
 * @returns {unknown}
```